_posts/2017-03-17-getting-rest-right.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: Getting REST right
+author: jake_hall
+---
+
+It seems everyone is building a REST API these days - this is great! It's not easy, but there are a few key things you can do to make it easy for your consumers of your API.
```

Comment:
Think it's a good one for another post? Trying to keep the length down and focussing mainly on having chosen to go with REST, how to do it as well as it can be done!

There are benefits to both, it depends what kind of data you're providing.